### PR TITLE
New content

### PR DESCRIPTION
**Pull request by mdeneva**

### CODE REVIEW

File: docs/reference/cluster-operator/examples.md [hunk 1]
```diff
@@ -161,6 +161,77 @@ Limiting Ondat can cause malfunction for IO to Ondat volumes, therefore
...
+           limits:
+             cpu: 100m
+             memory: 200Mi
+           requests:
+             cpu: 10m
+             memory: 50Mi
+```
+
+```yaml
+resources:
```
Comment (cvlc) [inline docs/reference/cluster-operator/examples.md:223]: Missing a `name:` here

File: docs/reference/cluster-operator/examples.md [hunk 2]
```diff
@@ -161,6 +161,77 @@ Limiting Ondat can cause malfunction for IO to Ondat volumes, therefore
 we do not currently recommend applying upper limits to resources for Ondat
```
Comment (cvlc) [inline docs/reference/cluster-operator/examples.md:161]: Maybe we should remove this, as it conflicts with the following content that provides limits.

File: docs/reference/cluster-operator/examples.md [hunk 3]
```diff
@@ -161,6 +161,77 @@ Limiting Ondat can cause malfunction for IO to Ondat volumes, therefore
...
 pods.
 
+We have the following limits for all of our components:
+
+```yaml
+name: provisioner
```
Comment (Arau) [inline docs/reference/cluster-operator/examples.md:167]: the indentation broke for the `name` fields.

**Review by Arau — CHANGES_REQUESTED** [review]
Thank you for the change. 

I believe @DavidMarchant made changes on the api-manager requests and limits, it is worth doublechecking that we set the latest ones.

**Review by Arau — APPROVED** [review]
Good to merge in my opinion. 

It would be great to merge before release 2.6 so we can incorporate in both 2.5 and latest for the next release.

**Comment by cannischan** [comment]
@Arau did we merge this at the end for 2.6.0 release branch?

**Comment by cannischan** [comment]
Hi @mdeneva , please fix the lint error and prepare this PR to be merged - this should go out together with 2.6.0 due to the operator limit that was added.

**Review by cvlc — APPROVED** [review]

**Pull request merged**